### DISCUSSION TITLE
Specify file header offset when using MagickImage

### DISCUSSION THREAD

**Discussion by avironman** [dlemstra/Magick.NET#1217] [Help]
Hello,
When loading a raw file using IM in cmd line, I'm used to write:
`magick -size 752x480+40 -depth 8 gray:inputfile.raw outputfile.tiff`

how can I do the same in .Net C# and specify the +40 offset when using Magick.Net instance creation of MagickImage
Here is my code :

```
private void ReadImagick(string Filename, int Width, int Height)
        {
            // Read image that has no predefined dimensions.
            var settings = new MagickReadSettings();
            settings.Width = Width;
            settings.Height = Height;
            settings.Depth = 8;
            settings.ColorType = ColorType.TrueColor;
            settings.Compression = CompressionMethod.NoCompression;
            settings.Format = MagickFormat.Gray;

            using (var image = new MagickImage(Filename, settings))
            {
                pictureBox1.Image = BitmapFromSource(image.ToBitmapSource<byte>());
            }
        }
```

Thanks

**Comment by dlemstra** [comment]
Can you share the input file so I can see what is happening?

**Reply by avironman** [reply]
Hello,
here is my raw file, first 40 bytes are header, after that, 752x480 pixels order in RG/GB
[inputfile.zip](https://github.com/dlemstra/Magick.NET/files/9096688/inputfile.zip)

**Reply by dlemstra** [reply]
I would advise you to read the file in a stream and then set the `Position` to `40`. There is no build in way to do this inside ImageMagick.
